bin/selection_utils.py

```diff
@@ -250,9 +230,8 @@ def trans_BA(BnA):
         #print(beta)
         #print(BnA)
         power_val = -beta*BnA
-        power_val[power_val < -125] = -125
-        #lambda_ = 2*(alpha + (2 - alpha) / (1 + Q*numpy.exp(-beta*BnA))) 
-        lambda_ = 2*(alpha + (2 - alpha) / (1 + Q*numpy.exp(power_val))) ##multiplied by 2 here 
+        power_val[power_val < -125] = -125  
```

Comment:
What's this line doing?

Reply:
Naveen (pasting from slack): I was getting underflow errors. So this takes the -beta*BnA values in the array power_val and changes them so if value < -125, it becomes -125. Ultimately, in the logistic equation for lambda, the Q*numpy.exp(power_val) goes to 0 if power_val is very negative.

So assigning the value -125 is functionally equivalent but gets rid of underflow values. That might be a hacky method, and the choice of -125 itself was arbitrary. Any other very low value that  prevents underflow values will do.